serverless.yml

```diff
@@ -54,10 +54,6 @@ functions:
           path: greenwood
           method: get
           cors: true
-      - http:
```

Comment:
This endpoint is no longer needed, and so is removed from the serverless template